airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/create_issues.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+import argparse
+import json
+import logging
+import os
+import subprocess
+import tempfile
+
+from definitions import GA_DEFINITIONS
+from jinja2 import Environment, FileSystemLoader
+
+TEMPLATES_FOLDER = "./templates/"
+COMMON_ISSUE_LABELS = ["area/connectors", "team/connectors-python", "type/enhancement", "test-strictness-level"]
```

Comment:
Note that we also have connector-specific labels (e.g. `connectors/source/youtube-analytics`) - we _may_ be able to infer these labels from the docker image name but I don't know if we're always following that pattern, so for these purposes it's probably ok to leave out.

Reply:
Yes I was able to infer the labels quite right, but I'm afraid some of these are not matching perfectly the usual naming rule. If a label does not exist the CLI fails. And there's no feature to list existing labels with the CLI.
So I thought it was good enough not to add the connector-specific label.